test/irmin-pack/test_flush_reload.ml

```diff
@@ -135,7 +128,10 @@ let test_one t ~(ro_reload_at : phase_flush) =
           | `After_dict -> aux S2_after_flush_dict
           | `After_suffix -> aux S3_after_flush_suffix
         in
-        let () = Store.S.X.Repo.flush_with_hook ~hook rw in
```

Comment:
since you are replacing the `*_with_hook` functions here, maybe remove them from store.ml too?

Reply:
Done with 6667f4f4e91668c14b218db528fe364ad5352369.